src/linuxkit_build.ml

```diff
@@ -385,14 +398,13 @@ module Result_cache = Cache.Make(Builder)
 
 type t = Result_cache.t
 
-let build t ~target src =
+let build t ~whitelist ~target src =
   let open! Term.Infix in
   Term.job_id >>= fun job_id ->
-  let key = { Builder.Key.
-              src;
-              target;
-            } in
-  Result_cache.find t job_id key
+  Term.target target >>= fun target_v ->
+  let key = { Builder.Key. src; target; whitelist; } in
+  let context = { Builder.job_id; target = target_v } in
```

Comment:
I don't think putting the target in the context makes sense either. Whether a PR is built does depend on the state of the PR, not just on the commit.